javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
```

Comment:
What if there was nothing stored in localStorage? This would replace your array with `null`, you should always double-check if there was actually a value stored in localStorage